packages/cli/test/shared/mocking.ts

```diff
@@ -11,3 +12,13 @@ export const mockInstance = <T>(
 	Container.set(serviceClass, instance);
 	return instance;
 };
+
+export const mockDBConnection = (entityClass: Function) => {
```

Comment:
1. Based on the name, as a caller I'd expect this to mock a `DataSource`. Maybe `mockEntityManager`? 
2. `Function` is triggering a lint warning. Also why `Function` and not something like `new () => {}`?